### ISSUE TITLE
paper for multi-output regression

### ISSUE THREAD

**Issue by iuiu34** [dmlc/xgboost#9671]
hi,
xgboost 2.0 added the very cool feature multioutput regression [link](https://xgboost.readthedocs.io/en/stable/tutorials/multioutput.html)

How to use it is clear with the docs. But how it works at statistical lvl no.

There is any paper for this? similar to the original, and very didactic, [one for xgboost](https://arxiv.org/pdf/1603.02754.pdf)?
giving and outline of pros and cons of one_output_per_tree  vs multi_output_tree ?

thanks

**Comment by trivialfis** [comment]
We are still working on it.

**Comment by trivialfis** [comment]
Closing in favor of https://github.com/dmlc/xgboost/issues/9043 . We will write some more document for the feature once the feature is complete.

**Comment by gspacone** [comment]
Is there a follow-up on this topic?

**Comment by hcho3** [comment]
Please see https://github.com/dmlc/xgboost/issues/9043
